backend/src/main/java/io/diveni/backend/controller/RoutesController.java

```diff
@@ -109,6 +113,75 @@ public ResponseEntity<Session> getExistingSession(@RequestParam("adminCookie") U
     }
   }
 
+  @GetMapping(value = "/data")
```

Comment:
I would suggest to create a separate controller `AnalyticsController` to include all mappings on `/data`. 
Also rename the URI to `/analytics` to be more specific